src/Compiler/CompilerUtils.cpp

```diff
@@ -126,14 +126,6 @@ static llvm::cl::opt<std::string> march("march",
     llvm::cl::value_desc("Target a specific architecture type"),
     llvm::cl::cat(OnnxMlirOptions), llvm::cl::ValueRequired);
 
-static llvm::cl::opt<OptLevel> OptimizationLevel(
```

Comment:
I think we should keep all options that are exposed to the user together. `onnx --help` should show them all under the same category (`OnnxMlirOptions`).  Is my understanding that in `OMOptions.cpp` we should have options that  "provide fine control on passes".  Perhaps you can make this option non-static and expose it to the accelerator via an extern declaration?

Reply:
I agree, options are now quite scattered. I think it requires more work to reorganize options, so I have no plan to do it in this patch.